sdk/webpubsub/Azure.Messaging.WebPubSub.Client/src/Protocols/WebPubSubJsonProtocol.cs

```diff
@@ -46,7 +46,7 @@ public override ReadOnlyMemory<byte> GetMessageBytes(WebPubSubMessage message)
         /// </summary>
         /// <param name="input">The serialized representation of the message.</param>
         /// <returns>A <see cref="WebPubSubMessage"/></returns>
-        public override WebPubSubMessage ParseMessage(ReadOnlySequence<byte> input)
+        public override IList<WebPubSubMessage> ParseMessage(ReadOnlySequence<byte> input)
```

Comment:
If we don't intend that callers should be able to modify this set, we should consider `IReadOnlyList<T>` as the return type for these.